itkwidgets/widget_viewer.py

```diff
@@ -10,6 +10,7 @@
 import time
 
 import itk
+import vtk
```

Comment:
We want to guard this import like this:

https://github.com/InsightSoftwareConsortium/itkwidgets/blob/582cc14f2de87054877adf9cc250bbee59b36ed4/itkwidgets/_transform_types.py#L18-L23

and make the implementation that needs it check `have_vtk`.